packages/backend/src/config/app.ts

```diff
@@ -80,21 +92,15 @@ export const Config = {
   trustProxy: envBool('TRUST_PROXY', false),
   redisUrl: envString('REDIS_URL', 'redis://127.0.0.1:6379'),
   redisTls: parseRedisTlsConfig(
-    envString('REDIS_TLS_CA_FILE_PATH', ''),
-    envString('REDIS_TLS_KEY_FILE_PATH', ''),
-    envString('REDIS_TLS_CERT_FILE_PATH', '')
+    process.env.REDIS_TLS_CA_FILE_PATH,
+    process.env.REDIS_TLS_KEY_FILE_PATH,
+    process.env.REDIS_TLS_CERT_FILE_PATH
   ),
-  ilpAddress: envString('ILP_ADDRESS', 'test.rafiki'),
-  ilpConnectorAddress: envString(
-    'ILP_CONNECTOR_ADDRESS',
-    'http://127.0.0.1:3002'
-  ),
-  instanceName: envString('INSTANCE_NAME', 'Rafiki'),
-  streamSecret: process.env.STREAM_SECRET
-    ? Buffer.from(process.env.STREAM_SECRET, 'base64')
-    : crypto.randomBytes(32),
-
-  useTigerbeetle: envBool('USE_TIGERBEETLE', false),
+  ilpAddress: envString('ILP_ADDRESS'),
+  ilpConnectorAddress: envString('ILP_CONNECTOR_ADDRESS'),
+  instanceName: envString('INSTANCE_NAME'),
+  streamSecret: Buffer.from(process.env.STREAM_SECRET || '', 'base64'),
+  useTigerbeetle: envBool('USE_TIGERBEETLE', true),
```

Comment:
maybe lets keep this as false as default, since the backend wouldn't be able to actually start without having the TB deployed?

Reply:
@sabineschaller we need you to make a decision here.

Reply:
I'd keep it as true. We do want to encourage the use of Tigerbeetle. It would be great if we had a meaningful error message when someone tried to start it with this set to true but Tigerbeetle not running.